lib/modules/manager/gradle/parser.ts

```diff
@@ -937,19 +1031,59 @@ function tryMatch({
   return null;
 }
 
-export function parseGradle(
+async function parseInlineScriptFile(
+  scriptFile: string,
+  variables: PackageVariables,
+  recursionDepth: number,
+  packageFile = ''
+): Promise<SyntaxHandlerOutput> {
+  if (recursionDepth > 0) {
+    logger.warn({ scriptFile }, `Max recursion depth reached`);
+    return null;
+  }
+
+  if (!regEx(/\.gradle(\.kts)?$/).test(scriptFile)) {
+    logger.warn({ scriptFile }, `Only Gradle files are eligible for inclusion`);
```

Comment:
```suggestion
    logger.warn({ scriptFile }, `Only Gradle files can be included`);
```

Would something like this work?

Reply:
Sure 👍